ui/src/app/pages/admin/admin-user-common.tsx

```diff
@@ -146,7 +147,7 @@ export const getUpdatedProfileValue = (
   if (!fp.isEqual(oldValue, updatedValue)) {
     return updatedValue;
   } else {
-    return null;
+    return undefined;
```

Comment:
This improves the JSON generated for the API call in updateAccountProperties().

Before we'd send null fields when values are missing.  Now we send no field at all.

Thanks for the suggestion @dmohs